en/flight_modes/follow_me.md

```diff
@@ -41,44 +41,47 @@ The following flight precautions should be observed:
     By *default* this is 8 metres/26 feet above the home (arming) position.
 - It is *safer* to manually fly to a safe height before engaging follow-me mode than to engage follow-me mode when landed (even though the mode implements auto take off).
 - Give your vehicle sufficient room to stop, especially when it is moving fast.
-  Many Android devices do not update their position very frequently, and autopilot estimations of the speed and direction can be inaccurate.
-- Be ready to take manual RC control if something goes wrong when using follow me mode for the first time.
-  The accuracy of positioning is dependent on the quality of the GPS used by target system.
-  If the GPS is not accurate, this will be reflected in follow me.
+- Be ready to switch back to Position mode, etc, if something goes wrong when using follow me mode for the first time.
+  If the target system (e.g. Android Phone)'s GPS is not accurate, this can affect the behavior.
 
-## Using Follow Me with QGroundControl

```

Comment:
Unless you are also covering MAVSDK in the instructions, it is probably better to note that this is using QGroundContol.

Note, I prefer the change you are suggesting, just needs to have some comments about what might be different when using it with MAVSDK.